build.zig

```diff
@@ -262,6 +262,14 @@ pub fn build(b: *std.Build) !void {
         if (config.emit_test_exe) b.installArtifact(test_exe);
         _ = try deps.add(test_exe);
 
+        // Only need ziglyph for tests
```

Comment:
We can move this to `SharedDeps` and check the `Step.Compile`'s `lib` property if its `.@"test".